apps/system/fxa/locales/fxa.en-US.properties

```diff
@@ -77,7 +77,7 @@ fxa-hello-user = Hello, {{email}}
 fxa-password-refresh = Please enter your Firefox Account password to continue.
 fxa-password.placeholder = Password
 fxa-forgot-your-password = Forgot your password?
-fxa-show-password = Show password
```

Comment:
as mentioned number of times earlier - it's deprecated to change existing l10n strings. If you need new text you shall create new l10n id, otherwise such changes wouldn't be detected by l10n team and translations wouldn't be updated at all.